docs/products/access/appendix/access-events.md

```diff
@@ -138,9 +138,17 @@
 | AUTHZ REQUEST SCOPE INVALID                                   | One or more of the requested scopes is not available for the specified client.                                                                                                                                       |
 | AUTHZ REQUEST NOT AUTHORIZED FOR SCOPE                        | The user is not authorized for the requested scope. See the [scope verification service](../topics/integration-extension/scope-verification/scope-verification.md) topic guide for more info.                        |
 | AUTHZ REQUEST GRANT CREATED                                   | An authorization code is created for the requested client and scopes.                                                                                                                                                |
+| AUTHZ REQUEST DEVICE GRANT CREATED                            | A device grant is created for the requested client and scopes.                                                                                                                                                       |
 | AUTHZ REQUEST INVALID CLIENT                                  | The `client_id` in the OAuth authorization request is invalid or unknown                                                                                                                                             |
 | AUTHZ REQUEST INVALID RESPONSE TYPE                           | The response type is not supported by the application. The OneWelcome Access only supports `response_type=code`.                                                                                                     |
 | AUTHZ REQUEST UNAUTHORIZED CLIENT                             | The client is not allowed to perform this action. Make sure the 'authorization code' flow is enabled for a used web client or 'user registration' for a mobile application.                                          |
+| AUTHZ REQUEST INVALID RESPONSE TYPE                           | The response type is not supported by the application. The Onegini Access only supports `response_type=code`.                                                                                                        |
```

Comment:
Isn't it a duplicate of event defined at line [142](https://github.com/onewelcome/identity-cloud-documentation/pull/343/files#diff-90679a7e3d0c5f9b84919587acce3c01fece27f20efa713836309c01b2450671L142)?